mongodbatlas/data_source_mongodbatlas_project_test.go

```diff
@@ -174,9 +175,10 @@ func TestAccProjectDSProject_limits(t *testing.T) {
 			{
 				Config: testAccMongoDBAtlasProjectDSUsingRS(testAccMongoDBAtlasProjectConfigWithLimits(projectName, orgID, []*projectLimit{})),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttrSet("data.mongodbatlas_project.test", "name"),
-					resource.TestCheckResourceAttrSet("data.mongodbatlas_project.test", "org_id"),
-					resource.TestCheckResourceAttrSet("data.mongodbatlas_project.test", "limits.0.name"),
+					testutils.CheckStateMatchesGoldenFile(
+						"data.mongodbatlas_project.test",
+						"../test/data/state-golden-files/project-ds/project-ds-limits.json",
+						[]string{".primary.attributes.org_id", ".primary.attributes.created", ".primary.attributes.name"}),
```

Comment:
Q: Why checking `.primary.attributes.created` now - time/dates tend to be a brittle test.

Also are you dropping the tests on `"limits.0.name"` ?

Reply:
This is actually just the opposite, the `CheckStateMatchesGoldenFile` allows the possibility of ignoring certain fields in the json. This relevant for values that change on each execution, like the `created` attribute that will always be different in value. All other fields will be used for comparison.